traitcuration/traits/datasources/zooma.py

```diff
@@ -29,26 +30,51 @@ def run_zooma_for_all_traits():
 
 def run_zooma_for_single_trait(trait):
     logger.info(f"Retrieving ZOOMA suggestions for trait: {trait.name}")
-    suggestion_list = get_zooma_suggestions_for_trait(trait)
-    # A set of suggested terms found in the query, used to exclude those terms from being deleted from the database
-    # when the delete_unused_mappings function is called.
-    suggested_terms = set()
-    for suggestion in suggestion_list:
+
+    datasource_suggestion_list = get_suggestions_from_datasources(trait)
+    ols_suggestion_list = get_suggestions_from_ols(trait)
+
+    high_confidence_term_iris = list()  # A list of terms with 'HIGH' mapping confidence
+    final_suggestion_iri_set = set()  # A set of the term iris which are kept from the ZOOMA queries
+
+    for suggestion in datasource_suggestion_list + ols_suggestion_list:
+        if len(suggestion["semanticTags"]) > 1:
+            logger.warn("Suggestion with combined terms found! Skipping suggestion...")
```

Comment:
Can we log the actual suggestion that shows combined terms. it might be useful for later.